Ports/diffutils/package.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port=diffutils
 version=3.7
+config_sub_path=build-aux/config.sub
```

Comment:
Please insert these after `use_fresh_config_sub` consistently, and not sometimes after `version` and sometimes not.